tas-installer/cmd/uninstall.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"log"
+	"securesign/sigstore-ocp/tas-installer/internal/uninstall"
+
+	"github.com/spf13/cobra"
+)
+
+var uninstallCmd = &cobra.Command{
+	Use:   "uninstall",
+	Short: "Removes installation of Trusted Artifact Signer",
+	Long:  `Removes installation of Trusted Artifact Signer (TAS) on a Kubernetes cluster.`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := uninstallTas(); err != nil {
+			log.Fatal(err)
+		}
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(uninstallCmd)
+}
+
+func uninstallTas() error {
+	if err := uninstall.HandleHelmChartUninstall(tasNamespace, tasReleaseName); err != nil {
+		log.Print(err.Error())
+	}
```

Comment:
This is a minor thing, but I've been trying to keep user-facing i/o out of `pkg` and have it all in `cmd`. This doesn't matter a whole lot right now, but I like to think of `pkg` as an API - the computer interface, and `cmd` as the user interface. Keeping the i/o out of `pkg` makes actually using it as an API down the road a little simpler. So... my recommendation would be to put the output text from `tas-installer/internal/uninstall/uninstall.go` here. E.g. 
```suggestion
	}
	// assumes that `HandleHelmChartUninstall()` returns an info string and an `info` var is set
	fmt.Printf("Uninstalled helm release: %s namespace: %s %s\n", tasReleaseName, tasNamespace, info)
```
This may be overkill for our purposes. Just a suggestion.

Reply:
updated